.run/Android (Google Play).dart.run.xml

```diff
@@ -1,8 +1,6 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Android (Google Play).dart" type="FlutterRunConfigurationType" factoryName="Flutter" folderName="Android">
     <option name="filePath" value="$PROJECT_DIR$/packages/app/lib/entrypoints/android/main_google_play.dart" />
-    <method v="2">
-      <option name="RunConfigurationTask" enabled="true" run_configuration_name="Enable scanner_mlkit dependency" run_configuration_type="ShConfigurationType" />
-    </method>
+    <method v="2" />
```

Comment:
Is that relevant here?

Reply:
Good that you catched that. That doesn't belong there